### ISSUE TITLE
Multilabel binary calsification can't force ouput to 0 and 1

### ISSUE THREAD

**Issue by bgcgarlipp** [dnouri/nolearn#179]
Hi 

I have a multiple label binary calcification that trains from my data. Input data is in format as described in 12.1 on http://scikit-learn.org/stable/modules/multiclass.html. The problem I am experiencing is that my output of my CCN is in floats between 0-1 and not 0 or 1 as I would expect. Here is my neural net

```python    
    net2 = NeuralNet(
        layers=[
            ('input', layers.InputLayer),
            ('conv1', layers.Conv2DLayer),
            ('pool1', layers.MaxPool2DLayer),
            ('conv2', layers.Conv2DLayer),
            ('pool2', layers.MaxPool2DLayer),
            ('conv3', layers.Conv2DLayer),
            ('pool3', layers.MaxPool2DLayer),
            ('hidden4', layers.DenseLayer),
            ('hidden5', layers.DenseLayer),
            ('output', layers.DenseLayer),
            ],
        # input layer
        input_shape=(None, 1, 96, 96),
        conv1_num_filters=32, conv1_filter_size=(3, 3), pool1_pool_size=(2, 2),
        conv2_num_filters=64, conv2_filter_size=(2, 2), pool2_pool_size=(2, 2),
        conv3_num_filters=128, conv3_filter_size=(2, 2), pool3_pool_size=(2, 2),
        hidden4_num_units=500, hidden5_num_units=500,
        output_num_units=46, 
        verbose=1,
        output_nonlinearity=nonlinearities.sigmoid,    
        update=nesterov_momentum,
        update_learning_rate=0.01,
        update_momentum=0.975,
        max_epochs=10,
        # #Here are the important parameters for multi labels
        regression=True,
        objective_loss_function=lasagne.objectives.binary_crossentropy,
     
        )
```

Is their anyway to force my output to 0 or 1 only?
Thanks in advance

**Comment by BenjaminBossan** [comment]
Just quickly check if it works if you remove the line 

`regression=True`

which should not be there for classification problems. Furthermore, try it without the line

`objective_loss_function=lasagne.objectives.binary_crossentropy,`

**Comment by dnouri** [comment]
Closing for lack of feedback.
